conan_build.py

```diff
@@ -5,7 +5,7 @@
     if os.getenv("CXX") == "g++-7":
         version = os.getenv("TRAVIS_TAG")
         if not version:
-            version = "dev"
+            version = "v0.2.8-p0"
```

Comment:
It was just a version mismatch.

Reply:
It has become `v0.2.9-p0` in the meantime. :slightly_smiling_face:

Reply:
no need to update the version here as this is managed by the CI, see *conan_build.py* in this repo